demo/site/src/app/api/site-preview/route.ts

```diff
@@ -1,19 +1,22 @@
-import { SitePreviewParams } from "@comet/cms-site";
+import { getValidatedSitePreviewParams, SitePreviewParams } from "@comet/cms-site";
+import createGraphQLClient from "@src/util/createGraphQLClient";
```

Comment:
The GraphQL client in the library was removed since implementing this, so you'll need to update this code.